config.toml

```diff
@@ -12,7 +12,7 @@
 
 # This is the config for the server
 [server]
-	address	= "localhost"
+	address	= "0.0.0.0"
```

Comment:
`localhost` and `127.0.0.1` don't work in Docker for me. Had to use `0.0.0.0`.

I'm OK with having a `config.toml.example`, but currently it doesn't throw an error if `config.toml` is missing, even though the code looks like it should, and then it starts the server with a `nil` address I guess?